book.md

```diff
@@ -620,14 +620,14 @@ fake call stack somewhere else, for example on the heap, and use a primitive
 that changes the stack pointer's value instead. This is known as stack
 pivoting\index{stack pivoting}.
 
-Note that this fake call stack contains NULL bytes, even without considering
+Note that this fake call stack contains NUL bytes, even without considering
 the exact values of the various return addresses included. An overflow bug that
 is based on a C-style string operation would not allow an attacker to replace
 the stack contents with this fake call stack in one go, since C-style strings
 are null-terminated and copying the fake stack contents would stop once the
```

Comment:
Thank you for taking the effort to correct this!
I'm assuming many readers might not be aware of the difference between NUL and NULL/null....
Maybe adding a link to https://en.wikipedia.org/wiki/Null-terminated_string might be worthwhile for those who want to know more?

Maybe putting the link on "null-terminated" might make most sense? :
```
are [null-terminated](https://en.wikipedia.org/wiki/Null-terminated_string) and copying...
```